extensions/wikia/TwitterTag/TwitterTagHooks.class.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class TwitterTagHooks {
+	
+	/*
+	 * onParserFirstCallInit
+	 *
+	 * Registers the <twitter> tag with the parser and sets its callback
+	 *
+	 * @param $parser - The parser
+	 * @return true
+	 */
+	static public function onParserFirstCallInit( Parser &$parser ) {
+		$parser->setHook( 'twitter', [ __CLASS__, 'parseTwitterTag' ] );
+		return true;
+	}
+	
+	/*
+	 * parseTwitterTag
+	 *
+	 * Parses the twitter tag. Checks to ensure the required attributes are there. 
+	 *   Then constructs the HTML after seeing which attributes are in use. 
+	 *
+	 * @param $input - not used
+	 * @param $args - The attributes to the tag in an assoc array
+	 * @param $parser - not used
+	 * @param $frame - not used
+	 * @return $html - The HTML for the twitter tag
+	 */
+	static public function parseTwitterTag( $input, array $args, Parser $parser, PPFrame $frame ) {
+		if( empty( $args['user'] ) ) {
```

Comment:
Conventions require a space between the `if` and the left parenthesis (`if(` -> `if (`).